### PR TITLE
Add more verifiable data

### PR DESCRIPTION
**Pull request by shc261392**
## Changed
- Update create_c2pa_manifest to include asset tree info and better capture details

### CODE REVIEW

File: src/numbers_c2pa/core.py [hunk 1]
```diff
@@ -95,15 +117,42 @@ def create_c2pa_manifest(
...
+                    '@context': {
+                        'EXIF': 'http://ns.adobe.com/EXIF/1.0/',
+                        'EXIFEX': 'http://cipa.jp/EXIF/2.32/',
+                        'dc': 'http://purl.org/dc/elements/1.1/',
+                        'rdf': 'http://www.w3.org/1999/02/22-rdf-syntax-ns#',
+                        'tiff': 'http://ns.adobe.com/tiff/1.0/',
+                        'xmp': 'http://ns.adobe.com/xap/1.0/'
+                    },
+                    'EXIF:GPSLatitude': format_geolocation(latitude),
+                    'EXIF:GPSLongitude': format_geolocation(longitude),
```
Comment (olgahaha) [inline src/numbers_c2pa/core.py:150]: I'm not sure whether EXIF:GPSLatitude and EXIF:GPSLongitude match the EXIF spec as described in the link [C2PA Specification](https://c2pa.org/specifications/specifications/1.3/specs/C2PA_Specification.html#_exif_information). Other than that, it looks good to me.

Comment (shc261392) [reply src/numbers_c2pa/core.py:150]: @olgahaha I took the field key and value format from [Adobe Verify Tool Doc](https://opensource.contentauthenticity.org/docs/verify/) and the content could be displayed on Verify Tool.

It's different from the EXIF spec (DMS format), but after a quick search I didn't find a battle-tested library with a convenient method to convert it. Concerning there might be some corner cases I'd prefer staying with current format for now. We can have a task for chaning the formatting to DMS format used in EXIF, write good test to ensure the conversion is correct, and also checking with Adobe c2pa team for their tendency on continuing to support decimal degree format.

Comment (shc261392) [reply src/numbers_c2pa/core.py:150]: @olgahaha Here is the taks for follow-up. https://app.asana.com/0/1201016280880508/1207538253204834/f Since other parts are ok I'll go ahead and merge and release the version.
